bedrock/mozorg/templates/mozorg/manifesto-base.html

```diff
@@ -0,0 +1,41 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{%- set_lang_files "main" -%}
```

Comment:
If the strings are only coming from main.lang, you don't need this line: [main.lang is a shared file](https://github.com/mozilla/bedrock/blob/master/bedrock/settings/base.py#L255).

Reply:
👍